### ISSUE TITLE
Error during version check at start-up

### ISSUE THREAD

**Issue by WoHuy** [rubberduck-vba/Rubberduck#6153]
**Rubberduck version information**
Version 2.5.9.6291
OS: Microsoft Windows NT 10.0.19045.0, x64
Host Product: Microsoft Office 2016 x86
Host Version: 16.0.5408.1001
Host Executable: WINWORD.EXE

**Description**
RD throws an error when performing a version check.

**To Reproduce**
1. Launch MS Word
2. Launch the VBE (Alt-F11)
3. Clic inside the VBE to activate it

**Expected behavior**
The VBE launches and shows me an empty workspace

**Screenshots**
Instead, I am receiving the below message
![image](https://github.com/rubberduck-vba/Rubberduck/assets/48773525/6e191731-2d11-4277-95fc-c5af2e2a5747)

**Logfile**
The error occurs at line 12 in this log file:
[RubberduckLog.txt](https://github.com/rubberduck-vba/Rubberduck/files/12660798/RubberduckLog.txt)

**Comment by retailcoder** [comment]
Probably related to whatever the problem is with the website at the moment; the URL being hit is responding with HTTP500 for some reason... ironically the Azure-hosted API is responding perfectly fine, but the live site isn't connected to it yet ...so I've tweaked the configuration on the live site to point it to Azure instead, and behold - the site is back online!

The legacy version check URL is now returning the expected version string again, thanks for reporting!

**Comment by WoHuy** [comment]
Hi, I just wanted to say I am still experiencing this issue.

**Comment by retailcoder** [comment]
So yeah, pointing the website's backend to the Azure-hosted API fixed the version check for legacy 2.x installs that request a dedicated URL to the website, which responds with a simple string value.
Since the GoDaddy-hosted API returns HTTP 503 for the endpoint being hit by the version check feature in v2.5.9, the feature isn't able to pull the latest version number. The linked pull request will fix the error-handling of the feature so it can blow up gracefully (merely logging a warning), but it'll still attempt to hit the GoDaddy-hosted API that's failing.

In other words there'll be 3 work-arounds once that PR is merged & released:
1. Disable version checking at startup from the settings dialog.
2. Manually edit the configuration to specify a working base URL for the API.
3. Do nothing: eventually I'll ditch GoDaddy and the api.rubberduckvba.com DNS will point to some Azure container app and it'll magically just start working again and stop logging that startup check warning one day.

**Comment by WoHuy** [comment]
In the version I use (2.5.9.6291), I was able to disable version checking already. Thanks for bringing that simple workaround to my attention, and for your ongoing commitment to this excellent tool!
